### PR TITLE
Fix: TextField.passwordMask doesn't work

### PR DESCRIPTION
**Pull request by infosia**
Fix for [TIMOB-19894](https://jira.appcelerator.org/browse/TIMOB-19894).

Test Case: Following script should show password-masked TextField.

```javascript
var win = Ti.UI.createWindow({
    backgroundColor: 'blue'
});
var field = Ti.UI.createTextField({ width:200, passwordMask:true, value:'pa33w0rd'});

win.add(field);
win.open();
```

### CODE REVIEW

**Comment by infosia** [comment]
test this please

**Comment by garymathews** [comment]
CR: **PASS**
FR: **PASS**

Waiting for Jenkins

File: Source/UI/src/TextField.cpp [hunk 1]
```diff
@@ -43,124 +62,234 @@ namespace TitaniumWindows
...
 		}
 
 		std::string TextField::get_value() const TITANIUM_NOEXCEPT
 		{
-			return TitaniumWindows::Utility::ConvertUTF8String(text_box__->Text);
+			if (text_box__) {
+				return TitaniumWindows::Utility::ConvertUTF8String(text_box__->Text);
+			} else if (password_box__) {
+				return TitaniumWindows::Utility::ConvertUTF8String(password_box__->Password);
+			}
```
Comment (garymathews) [inline Source/UI/src/TextField.cpp]: Add a default return value?

Comment (infosia) [inline Source/UI/src/TextField.cpp]: Oops yes. Nice catch.

**Comment by garymathews** [comment]
CR: **PASS**

**Pull request merged**